croaring/Cargo.toml

```diff
@@ -25,7 +25,7 @@ roaring = "0.10"
 criterion = { version = "0.5", features = ["html_reports"] }
 
 [dependencies]
-ffi = { package = "croaring-sys", path = "../croaring-sys", version = "~4.1.1" }
+ffi = { package = "croaring-sys", path = "../croaring-sys", version = "~4.1.2" }
```

Comment:
I see CI failed because of this dependency bump. Perhaps we shouldn't increase versions in the PRs, what do you think? Basically we can merge any amount of changes and then do a release commit [like this](https://github.com/RoaringBitmap/croaring-rs/commit/c7563ff31a2f83cb01b4ce061275e44b31fbb85c). Before release commit gets pushed to github you'd need to publish `croaring-sys` and then `croaring` crates of new versions. After release commit gets pushed you can do a Github release of a new version. That was the action sequence I always used. How does that sound?

Reply:
Hmm. I'm realizing with the changes from #149 to actually test `croaring-rs` against the real minimal versions, that test will always fail if we both update `croaring-sys` and `croaring-rs` to use the new functions at the same time.

Not sure what the best way forward is: could always split into two PRs, croaring-sys only, then push a new version of it, and then a PR to use the new functions after the sys crate is published. That seems a bit of overhead though.